src/components/Carousel/styled.ts

```diff
@@ -4,7 +4,7 @@ import Icon from '@/components/Icon';
 import { styled } from '@/styles';
 
 const BUTTON_SIZE = 42;
-export const CAROUSEL_GUTTER_WIDTH = 12;
+export const CAROUSEL_GUTTER_WIDTH = 14;
```

Comment:
on designs it was 14, I assume this has to do with the borders as well

Reply:
this was made `12` based on earlier feedback
https://www.notion.so/voiceflow/Chat-UI-Kit-c60c816c69ba43a59774f024608b7a4d#5c366c055327422aa2154378a82a16f7

it doesn't include the border which adds 1px on either side
[accounted for in the card width during scroll animations though](https://github.com/voiceflow/react-chat/blob/master/src/components/Carousel/constants.ts#L5)

Reply:
I think you might actually want to leave it as 12? it seems like the last card is slightly off
not 100% if this is the cause, but something is throwing off the math a bit
<img width="280" alt="Screen Shot 2022-09-30 at 5 11 52 PM" src="https://user-images.githubusercontent.com/3784470/193356997-b04b6b7d-3b8f-4c87-aa73-928c70f58bd4.png">

Reply:
I have a fix here: https://github.com/voiceflow/react-chat/pull/6#discussion_r985017116